manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -1622,9 +1622,12 @@ spec:
       patch:
         operation: INSERT_BEFORE
         value:
-          name: envoy.filters.network.metadata_exchange
-          config:
-            protocol: istio-peer-exchange
+          name: metadata_exchange
```

Comment:
should we name this network_metadata_exchange to separate it from HTTP WASM Metadata exchange?

Reply:
I didn't see it overlapping with other one, so probably doesn't matter much since the name could be used twice.